docker-storage-setup.sh

```diff
@@ -1049,8 +1049,10 @@ get_docker_root_dir(){
                 ;;
         esac
     done
-    if ! DOCKER_ROOT_DIR=$(realpath -m $path);then
-      Fatal "realpath failed on $path"
+    if [ ! -z "$path" ];then
```

Comment:
@rhatdan Sorry I am not following. If $path is not empty, why would we return ? If $path is not empty, we should get the realpath: $(realpath -m $path) ?